src/main/java/org/folio/rest/impl/ContactsAPI.java

```diff
@@ -58,14 +58,14 @@ public void getVendorStorageContactPersons(String query, int offset, int limit,
           .setLimit(new Limit(limit))
           .setOffset(new Offset(offset));
 
-        PostgresClient.getInstance(vertxContext.owner(), tenantId).get(CONTACT_PERSON_TABLE, ContactPerson.class, fieldList, cql,
+        PostgresClient.getInstance(vertxContext.owner(), tenantId).get(CONTACT_PERSON_TABLE, Contact.class, fieldList, cql,
```

Comment:
This code can be simplified via PgUtils... see https://github.com/folio-org/mod-orders-storage/pull/82 for details.  Whether you address this as part of this PR or not is up to you.  I plan on creating a separate story to make these changes across the rest of this module.